### ISSUE TITLE
BTRFS support in fsType

### ISSUE THREAD

**Issue by mikroskeem** [openebs/zfs-localpv#169]
**Describe the problem/challenge you have**
I'm deploying Concourse CI with zfs-localpv and for optimal performance, I would like to use `btrfs` zvols (alternatives aren't really viable - overlay does not work on top of ZFS, naive is very slow and btrfs loopback device leaks /dev/loop* devices)

**Describe the solution you'd like**
Adding tools for creating btrfs formatted zvols

**Anything else you would like to add:**

**Environment:**
(irrelevant)

**Comment by pawanpraka1** [comment]
@mikroskeem thanks for raising this issue. btrfs use case is new to us. Can you elaborate a little bit about your use case why btrfs?  Why not use ext4 or zfs which will bind mount the volumes?
```
apiVersion: storage.k8s.io/v1
kind: StorageClass
metadata:
  name: openebs-zfspv
parameters:
  fstype: "zfs"
  poolname: "zfspv-pool"
provisioner: zfs.csi.openebs.io
```

Would you like to add your use case in our ADOPTERS.md file. It will help us out when working with CNCF. You can put a comment describing your use case here : https://github.com/openebs/openebs/issues/2719.

**Comment by mikroskeem** [comment]
I'm only using btrfs zvols, because Concourse performs best using btrfs on a disk (ignoring the fact that zfs adds a bit overhead, but still better than alternatives) and does not support zfs directly sadly.

About ADOPTERS document, I'll add my use-cases when I have more time :)

**Comment by pawanpraka1** [comment]
@mikroskeem I have raise the PR to add btrfs support : https://github.com/openebs/zfs-localpv/pull/170. Will be available for in v0.9 release (planned on 15th July).

**Comment by mikroskeem** [comment]
Amazing, thank you.
